### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by themacexpert**
Kevin does know bootstrap

### CODE REVIEW

File: assets/static/js/buzzer.js [hunk 1]
```diff
@@ -127,10 +127,11 @@ function repopulateTable(activeParties) {
     htmlStr += "<td>" + parseEstimatedWait(activeParties[i].WaitTimeExpected) + "</td>";
     if (activeParties[i].PhoneAhead) {
       htmlStr += "<td><span class=\"glyphicon glyphicon-earphone\"></span></td>";
+      htmlStr += '<td><div class="btn-toolbar"><button class="btn btn-default buzz-button" type="button">Assign Buzzer</button><button class="btn btn-default delete-party-button" type="button">Delete</button></div></td>';
     } else {
       htmlStr += "<td><span class=\"glyphicon glyphicon-user\"></span></td>";
+      htmlStr += '<td><div class="btn-toolbar"><button class="btn btn-default buzz-button" type="button">Buzz!</button><button class="btn btn-default delete-party-button" type="button">Delete</button></div></td>';
```
Comment (jfeiber) [inline assets/static/js/buzzer.js:133]: Can you change this to the way it was before? You just added a duplicate line

Comment (jfeiber) [reply assets/static/js/buzzer.js:133]: jk

**Comment by jfeiber** [comment]
:+1:

**Comment by themacexpert** [comment]
This fixes issues #66 and #100

**Pull request merged**